tests/connection/mod.rs

```diff
@@ -21,30 +21,46 @@
 use litep2p::{
     config::ConfigBuilder,
     crypto::ed25519::Keypair,
-    error::{AddressError, Error},
+    error::Error,
     protocol::libp2p::ping::{Config as PingConfig, PingEvent},
-    transport::{
-        quic::config::Config as QuicConfig, tcp::config::Config as TcpConfig,
-        websocket::config::Config as WebSocketConfig,
-    },
+    transport::tcp::config::Config as TcpConfig,
     Litep2p, Litep2pEvent, PeerId,
 };
 
+#[cfg(feature = "websocket")]
```

Comment:
nit: It might be worth having this module structured as:
- `mod.rs` to bridge the `quic.rs` `websocket.rs` and `tcp.rs` depending on feature flags; and here we can have the common logic as well
- then have the tests moved to individual files so that the only `cfg` we state is in mod.rs


```
#mod.rs

#[cfg(feature = "quic")]
use quic.rs
...
```

We can also handle this later since I expect to run into some edge-cases along the way which may involve a closer refactor

Reply:
I was thinking about putting quick/websocket/webrtc tests into dedicated files, but it looks like the tests only differ in the transport used, and belong better to the connection/protocol test files. I.e., it would be easier to have these tests along each other in the same file. I will think if there is another alternative to having `cfg` spread over the files. Let me know if you have anything in mind